test/EFCore.PG.FunctionalTests/ExecutionStrategyTest.cs

```diff
@@ -105,82 +124,81 @@ void Test_commit_failure(bool realFailure, Action<TestNpgsqlRetryingExecutionStr
+                realFailure, (e, db) => e.ExecuteInTransactionAsync(
+                    ct => db.SaveChangesAsync(false, ct),
+                    ct => db.Products.AsNoTracking().AnyAsync(),
+                    CancellationToken.None));
+
+            await Test_commit_failure_async(
+                realFailure, (e, db) => e.ExecuteInTransactionAsync(
+                    db,
+                    async (c, ct) => { await c.SaveChangesAsync(false, ct); },
+                    (c, ct) => c.Products.AsNoTracking().AnyAsync(),
+                    CancellationToken.None));
+
+            await Test_commit_failure_async(
+                realFailure, (e, db) => e.ExecuteInTransactionAsync(
+                    db,
+                    (c, ct) => c.SaveChangesAsync(false, ct),
+                    (c, ct) => c.Products.AsNoTracking().AnyAsync(),
+                    CancellationToken.None));
+
+            await Test_commit_failure_async(
+                realFailure, (e, db) => e.ExecuteInTransactionAsync(
+                    () => db.SaveChangesAsync(false),
+                    () => db.Products.AsNoTracking().AnyAsync(),
+                    IsolationLevel.Serializable));
+
+            await Test_commit_failure_async(
+                realFailure, (e, db) => e.ExecuteInTransactionAsync(
+                    async ct => { await db.SaveChangesAsync(false, ct); },
+                    ct => db.Products.AsNoTracking().AnyAsync(ct),
+                    IsolationLevel.Serializable,
+                    CancellationToken.None));
+
+            await Test_commit_failure_async(
+                realFailure, (e, db) => e.ExecuteInTransactionAsync(
+                    ct => db.SaveChangesAsync(false, ct),
+                    ct => db.Products.AsNoTracking().AnyAsync(ct),
+                    IsolationLevel.Serializable,
+                    CancellationToken.None));
+
+            await Test_commit_failure_async(
+                realFailure, (e, db) => e.ExecuteInTransactionAsync(
+                    db,
+                    async (c, ct) => { await c.SaveChangesAsync(false, ct); },
+                    (c, ct) => c.Products.AsNoTracking().AnyAsync(ct),
+                    IsolationLevel.Serializable,
+                    CancellationToken.None));
+
+            await Test_commit_failure_async(
+                realFailure, (e, db) => e.ExecuteInTransactionAsync(
+                    db,
+                    (c, ct) => c.SaveChangesAsync(false, ct),
+                    (c, ct) => c.Products.AsNoTracking().AnyAsync(ct),
+                    IsolationLevel.Serializable,
+                    CancellationToken.None));
         }
 
-        async Task Test_commit_failure_async(bool realFailure, Func<TestNpgsqlRetryingExecutionStrategy, ExecutionStrategyContext, Task> execute)
+        private async Task Test_commit_failure_async(
+            bool realFailure, Func<TestNpgsqlRetryingExecutionStrategy, ExecutionStrategyContext, Task> execute)
```

Comment:
Did Rider put on the second line it?

Reply:
Copy-pasted from EF Core. But we also have a maximum column settings that causes Rider to break lines - I don't always respect it.